src/OrleansProviders/Streams/Common/PooledCache/ChronologicalEvictionStrategy.cs

```diff
@@ -84,6 +84,16 @@ public void PerformPurge(DateTime nowUtc)
             this.periodicMonitoring?.TryAction(nowUtc);
         }
 
+        /// <inheritdoc />
+        public virtual bool ShouldPurge(ref CachedMessage cachedMessage, ref CachedMessage newestCachedMessage, DateTime nowUtc)
```

Comment:
Unless it needs to be public, suggest making it a protected virtual.  Specializations can override, but still an implementation detail of this strategy.

Reply:
Okay, will make it protected virtual, somehow that skipped my mind that a "protected virtual" would also work.